tests/parser.tests

```diff
@@ -18,25 +18,25 @@ filemode : true
 
 Recursive row variable in variant [1]
 sig f : ([|(mu a. Op:() { |a}-> ())|]) {}-> () fun f(_) { () } f
-stdout : fun : ([||(mu a . Op:() {|a}-> ())|]) {}-> ()
+stdout : fun : ([|(mu a . Op:() { |a}-> ())|]) {}-> ()
 
 Recursive row variable in variant [2]
 sig tsize : ([|( mu t. Node:([|t|],[|t|])|Leaf)|]) ~> Int fun tsize(t) { switch (t) { case Leaf -> 1 case Node(l,r) -> 1 + tsize(l) + tsize(r) } } tsize
-stdout : fun : ([||(mu a . Leaf|Node:([||a|], [||a|]))|]) ~> Int
+stdout : fun : ([|(mu b . Leaf:(),Node:([|b|], [|b|]))|]) ~> Int
```

Comment:
I think this printing is incorrect. It is a recursive row variable in a **variant** row. Thus the fields should be `vfields`, i.e. the separator is `|`.

Reply:
You are right, fixing this.